### ISSUE TITLE
Demo with Vince

### ISSUE THREAD

**Issue by swashcap** [trendscenter/coinstac#90]
Demo the UI with Vince. Make it great!

**Comment by swashcap** [comment]
Vince, @rssk and I demoed the built application today on Windows and MacOS. Some comments, via Vince:
- Computations and data should be separated: multishot is closely coupled to FreeSurfer data at the moment. What if we want to analyze a different type, say heart rate measurements? (See #96)
- It would be nice to have some UI that describes project status: perhaps a dashboard of the current computations, who are you waiting on, what files are active in a computation, which consortium, etc. (See #97)
- We need more user-friendly output with the capability for nice interpretations and visualizations. (See #12)
